robotbase/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,18 +21,18 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.RamseteCommand;
 
-//import com.systemmeltdown.robot.commands.ShootCommand;
-//import com.systemmeltdown.robot.subsystems.IntakeSub;
-//import com.systemmeltdown.robot.subsystems.ShooterSubsystem;
-//import com.systemmeltdown.robot.subsystems.StorageSubsystem;
+import com.systemmeltdown.robot.commands.ShootCommand;
```

Comment:
Comment out all subsystems that aren't currently set up on the bot, and all of their uses on RobotContainer